### DISCUSSION TITLE
Whitelist labels and syntax example of remove VRL function

### DISCUSSION THREAD

**Discussion by youngpabl0** [vectordotdev/vrl#774] [Q&A]
### A note for the community

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Proposal

Hi everyone, i want to clarify how to use a one VRL remap function: remove
My example: i'v got my logs from docker_local and send them to elasticsearch, and before sending i want to filter with whitelist my labels from containers, and i dont find any better solution is just use remove VRL function
With dat example i have 2 questions:
1 - help me to understand syntax of remove func please? `remove!(value: .label.org, path: ["*"])` doesnt work at all, i see in fields all `.label.org.*` labels, i'v look in documentation 10 times and examples in documentation are very unclear and synthetic (foo:bar and etc), can anyone please share real life examples or help with syntax in this thread?
2 - may be it's good feature to whitelist fields in remap section, cause it will guard future logs from document exceptions or other errors from elastic

thanx everyone

### Version

0.36.0-debian

**Comment by jszwedko** [comment]
You may be interested in this example: https://github.com/vectordotdev/vrl/blob/main/lib/tests/tests/rfcs/8381/re_introduce_previous_only_fields_functionality_using_iteration.vrl

The `remove` function only takes actual path segment names, not wildcards.
